applications/blockchainia.md

```diff
@@ -0,0 +1,139 @@
+
+
+ Development Roadmap :nut_and_bolt:
+
+ ### Overview
+
+ - **Total Estimated Duration:** Continuous
+ - **Full-Time Equivalent (FTE):**  2
+ - **Total Costs:** $25,000
+
+ ### Milestone 1  — Basic MVP
+
+ - **Estimated duration:** 13 weeks development 
+ - **FTE:**  .97 FTE
+ - **Costs:** $25,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. Unit tests will be written for relevent game server functionality. Unit tests will be written for the DeathToll pallet. In each guide (in each project's Github repo), we will describe how to run these tests. Integration testing will be enabled through game functionality (i.e, Given an enemy is eliminated on server, the proper state functions are executed on chain, and the leaderboard updates through the in-game chain browser) |
+| **0d.** | Docker | We will provide Dockerfiles that can be used to run and test all the functionality delivered with this milestone. This will include a deployable "Full Node" that encases both a game server and substrate node. These will interact with our game client and substrate parachain respectively. A successful MVP will allow a game client to register events on the game server (Server-Authoritative model), which is then written to the parachain ledger. Upon completion of the initial game "level", an "accolade" NFT will be deposited via ink! smart contract to the wallet that completed the required in-game tasks. |
+| 1. | Social Media | We will deploy a social media marketing campaign upon the successful development and testing of an MVP. This will include a social media campaign, building a discord community of evangelists, and marketing materials that describe our project to potential stakeholders and various customers of our ecosystem. The first milestone will involve hitting 1000 followers, which will enable us to involve our community from Milestone 2 forward. 
+| 2. | Substrate module: DeathToll Game Events | The DeathToll Game Events Substrate module will add functionality for starting/ending a "match"", start and end a "round", eliminate a player, score a point, complete a "task". Tasks examples include "move through a defined space", eliminate an AI enemy, retrieve an item, etc. This deliverable will include unit tests and instructions in the git repository. It is highly likely that we will break this into two pallets, one for core gaming functionality, and one specific to the needs of the DeathToll game-type. 
```

Comment:
I have not started the implementation for the DeathToll game events pallet, although we are ready to. We have decided on the minimum features required to create the MVP that will get us the appropriate data needed from our planned market research. The core functionality to run our MVP FPS game on a multiplayer pay-for-play arcade basis will be included in the DeathToll pallet. Initially we will include all of the functionality to run the game in one pallet, but as we expand features, it may make sense to break them up into multiple open-source pallets. The MVP features are listed and detailed documentation will be completed as part of this deliverable.